codes/quantum/qubits/quantum_secret_sharing.yml

```diff
@@ -0,0 +1,108 @@
+
+  # This field that starts a list of strings, indexed by a dash; do not leave it
+  # empty. If empty, comment out.
+  encoders: 
+    - 'Uses a quantum authentication scheme, which is a keyed system in which a valid state has high fidelity, and a classical secret sharing scheme. '
+ 
+# general_gates:
+#- 'Universal gate set achieved by either additional ... gate.'
+#- 'Magic-state distillation protocols'
+#- 'kth Clifford hierarchy gates obtained by ... circuits'
+
+  decoders: 
+    - 'Decoding is analagous to reconstruction in a secret sharing scheme.'
+    - 'Decoding done in polynomial time. Only operations required are verification of quantum authentication, which is a pair of polynomial time quantum algorithms that
+	  check if the fidelity of the received state is close to 1,  and erasure correction for a stabilizer code, which involves solving a system of linear equations.' 
+
+#  fault_tolerance:
+#    - 'Transversal gates are fault-tolerant w.r.t. ... noise \cite{doi:ok-paper}'
+#    - 'Other fault-tolerant gadgets (measurements, encoders, error correcting steps)'
+#    - 'Noise-model-preserving gadgets, noise-biased gates, fault-tolerant flag error correction'
+#    - 'Pieceable fault tolerance.'
+
+#  threshold: 
+#    - '2\% error-correction threshold  ... with perfect ... decoder \cite{arxiv:another-paper}.'
+#   - '0.3\% error-correction threshold ... with noisy MWPM decoder \cite{doi:good-paper}.'
+#   - '\(10^{-5}\) computational threshold using concatenated scheme under ... noise with overhead of ... '
+#  - 'Asymptotic lower bound of ... on threshold with ... noise with ... overhead.'
+#- 'Thresholds using magic-state distillation protocols assuming perfect Clifford gates.'
+  
+# Propose a field that you feel is applicable to a large and interesting class of codes
+#some_other_field:
+  # - 'detail'
+
+realizations:
+  # List and explain the different "domains" of realizations in list items.
+  # Group closely related realizations into a single item. It's OK to have a
+  # single list item. Each item should more or less correspond to what would be
+  # said in a paragraph.
+  - 'This code requires a small logical subspace and large registers that contain both classical and quantum information. Given this constraint, this code is not useful for practical error correction problems, but demonstrates the power of approximate quantum error correction. '
+  - 'The code can be viewed as an t-error tolerant secret sharing scheme'
+
+notes:
+  - 'Bounds on n,k, or d for this class.'
+  - 'Connections to physics; based on \(H^4(\mathcal{BG},\mathbb{Z})\)'
+  - 'Can reference other codes, but try to put such connections in the relations fields.'
+  - 'Anything noteworthy that''s not above or below.'
+
+relations:
+  parents:
+    - code_id: approximate_quantum
+      detail: 'approximate_quantum codes include this family  of codes.'
+
+  cousins:
```

Comment:
They use classical info; is it encoded in any specific way? If so, please put down a cousin for that code.